app/helpers/posts_helper.rb

```diff
@@ -0,0 +1,2 @@
+module PostsHelper
```

Comment:
delete this file because not use

Reply:
![Uploading Screenshot from 2021-08-25 13-59-27.png…]()